### PR TITLE
[New] First Time Seen NewCredentials Lgon Process

### PR DESCRIPTION
**Pull request by Samirbous**
token make commands in popular HKTL/testing frameworks allow to duplicate or create a token and use it for netonly activities, below example of matches for both CS/Havoc : 

image

This rule uses New Term rule type to identify rare/unusual processes.

image

### CODE REVIEW

File: rules/windows/privilege_escalation_newcreds_logon_rare_process.toml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+author = ["Elastic"]
+description = """
+Identifies a new credentials logon type performed by an unusual process. This may indicate the existence of an access token
+forging capability that are often abused to bypass access control restrictions.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-system.*", "logs-windows.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "First Time Seen NewCredentials Lgon Process"
```
Comment (Aegrah) [inline rules/windows/privilege_escalation_newcreds_logon_rare_process.toml:19]:
```suggestion
name = "First Time Seen NewCredentials Logon Process"
```

File: rules/windows/privilege_escalation_newcreds_logon_rare_process.toml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+risk_score = 47
+rule_id = "e468f3f6-7c4c-45bb-846a-053738b3fe5d"
+severity = "medium"
+references = ["https://www.elastic.co/pt/blog/how-attackers-abuse-access-token-manipulation"]
+tags = ["Domain: Endpoint", "OS: Windows", "Use Case: Threat Detection", "Tactic: Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+event.category:"authentication" and host.os.type:"windows" and winlog.logon.type:"NewCredentials" and (winlog.event_data.LogonProcessName:(Advapi* or "Advapi  ")
```
Comment (Aegrah) [inline rules/windows/privilege_escalation_newcreds_logon_rare_process.toml]:
```suggestion
event.category:"authentication" and host.os.type:"windows" and winlog.logon.type:"NewCredentials" and winlog.event_data.LogonProcessName:(Advapi* or "Advapi  ")
```

File: rules/windows/privilege_escalation_newcreds_logon_rare_process.toml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+[rule.threat.tactic]
+id = "TA0004"
+name = "Privilege Escalation"
+reference = "https://attack.mitre.org/tactics/TA0004/"
+
+
+[rule.new_terms]
+field = "new_terms_fields"
+value = ["process.executable"]
```
Comment (Aegrah) [inline rules/windows/privilege_escalation_newcreds_logon_rare_process.toml:53]: Any reason not to add host.id or agent.id in there and bump up min stack?

**Review by Aegrah — APPROVED** [review]

File: rules/windows/privilege_escalation_newcreds_logon_rare_process.toml (hunk 2 above)
Comment (Aegrah) [reply rules/windows/privilege_escalation_newcreds_logon_rare_process.toml]: This was fixed while I was reviewing.

File: rules/windows/privilege_escalation_newcreds_logon_rare_process.toml (hunk 3 above)
Comment (Samirbous) [reply rules/windows/privilege_escalation_newcreds_logon_rare_process.toml:53]: process.executable is enough

**Review by terrancedejesus — APPROVED** [review]
Nice addition!

**Pull request merged**